### DISCUSSION TITLE
Obsidian Highlightr

### DISCUSSION THREAD

**Discussion by MuRiMo07** [BryanJenksCommunity/FAQ#150] [Q&A]
Hello

Been using the Bryans Gruvbox theme. The plugin highlightr does not seem to work properly. I can only highlight with the orange color. All the other highlight colors default to orange.

**Comment by tallguyjenks** [comment]
depending on when you got the theme file, I'm always updating it, if your theme file does not have a section in it numbered `2.3.9` corresponding to the highlight colors, then that may be 1 reason why it isn't working
